python/perspective/perspective/table/_accessor.py

```diff
@@ -128,9 +130,13 @@ def marshal(self, cidx, ridx, type):
         Returns:
             object or None
         '''
-        val = self.get(cidx, ridx)
+        column_name = self._names[cidx]
```

Comment:
the order of the dataset in the row might not be the same order as the `_names` attribute - this caused issues with partial updates using `__INDEX__` as `__INDEX__` would be at the end but it would be in the first column of the update data.